test/units/testsuite-02.sh

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
-#set -ex
-#set -o pipefail
+set -eux
+set -o pipefail
 
 NPROC=$(nproc)
 MAX_QUEUE_SIZE=${NPROC:-2}
-IFS=$'\n' TEST_LIST=($(ls /usr/lib/systemd/tests/test-*))
+mapfile TEST_LIST < <(find /usr/lib/systemd/tests/ -maxdepth 1 -type f -name "test-*")
```

Comment:
Whoah, what does `< <` even mean?

Reply:
`<(echo x)` creates a transient `/dev/fd/xx` file descriptor which contains output of the enclosed command. `<` then uses it as an input file.